tpm2/tpm2.go

```diff
@@ -705,11 +721,7 @@ func PolicySecret(rw io.ReadWriter, entityHandle tpmutil.Handle, entityAuth Auth
 
 	// Tickets are only provided if expiry is set.
 	if expiry != 0 {
-		tk, err := decodeTicket(bytes.NewBuffer(resp))
```

Comment:
Am I correct in understanding this didn't work in the current implementation, because it was missing decode for the `TPM2B_TIMEOUT` parameter?